bin/dune

```diff
@@ -0,0 +1,23 @@
+(executable
+ (public_name ocamlformat)
+ (package ocamlformat)
+ (modules ocamlformat)
+ (flags (:standard -open Import))
+ ;;INSERT_BISECT_HERE;;
```

Comment:
You need to substitute that in `tools/bisect.sh`.
(the `sed` command take only one file so you'll have to call it in a loop)

Reply:
`sed` seems to accept multiple files?

Reply:
It seems yes. My bad.
That's working.